src/plugins/cosmos/components/modals/Send/views/Confirm.tsx

```diff
@@ -0,0 +1,162 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
```

Comment:
This module is a copy of the existing `<Confirm />` with the addition of:
https://github.com/shapeshift/web/pull/1238/files#diff-1d0d4750995979e616dce4e87f33fa932399a6e7f908670a3981af2a4a18a8bfR99-R108

The reason for the duplication is that it consumes the Cosmos `SendInput` which is different for the existing one, and trying to use both `ensName` and `memo` on the existing `<Confirm />` would make us fight with types and is also blending too much chain-specific stuff in a general-purpose component, while chain-specific logic should live in plugins.